test/acceptance/features/steps/steps.py

```diff
@@ -466,3 +468,10 @@ def validate_secret_empty(context, secret_name):
                       ignore_exceptions=(json.JSONDecodeError,))
     except polling2.TimeoutException:
         pass
+
+
+@given(u'"{app_name}" is deployed from image "{image_ref}"')
+def create_deployment(context, app_name, image_ref):
+    app = App(app_name, context.namespace.name, image_ref)
+    if not app.is_running():
+        assert app.install() is True, "Failed to create deployment."
```

Comment:
after installing we should check once again if it is running, but this time we should wait until this happens or the timeout occurs.

Reply:
This case is handled by install function implicitly so we won't need to check it again.